hack/images/xslttransform/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM debian:11
```

Comment:
```suggestion
FROM debian:11-slim
```

`debian:11-slim` provides a slimmer base, this change yields a significantly smaller images

Reply:
done